doc/install.sgm

```diff
@@ -1,123 +1,98 @@
-                <application>PostgreSQL</application>'s source tree
-                (<filename>POSTGRESQL_SRC/src/contrib</filename>).
-                Then, change into <filename>POSTGRESQL_SRC/src/contrib</filename>.
-                If the sources are not yet installed and the directory
-                <filename>pg_sphere</filename> does not exist, take the
-                gzipped <application>pgSphere</application> sources ( e. g.,
-                <filename>pg_sphere_xxx.tgz</filename> ) and run:
-            </para>
-            <programlisting>
-<![CDATA[shell> tar -xzf path/to/pg_sphere_xxx.tgz]]>
-            </programlisting>
-            <para>
-                Now, change into the <filename>pg_sphere</filename>
-                directory and run :
-            </para>
-            <programlisting>
+  <para>
+   &pgsphere; is not the part of the &postgresql; software. You can download
+   the latest release sources from the
+   <ulink url="&pgsphereurl;">&pgsphere; Releases page</ulink>.
+   The sources can also be downloaded by cloning the repository with the
+   appropriate release tag. The master branch is intended for development
+   use and may contain the code in a transitional state. It is not recommended
+   for use in production.
+  </para>
+ </sect1>
+
+ <sect1 id="install-build">
+  <title>Install</title>
+  <para>
+   It is assumed that &postgresql; is already installed. Depending on the
+   system configuration, superuser (root) access rights may be required to
+   complete the installation.
+  </para>
+
+  <para>
+   The installation script uses &pg_config; utility. Make sure that the
+   environment variable PATH includes path to &pg_config; utility. The path
+   to &pg_config; can be also specified in make command:
+   make&nbsp;PG_CONFIG=/path/to/pgconfig.
+  </para>
+
+  <para>Unpack the downloaded archive and enter the directory:</para>
+  <programlisting>
+<![CDATA[shell> tar -xzf path/to/pgsphere-X.X.X.tgz]]>
+<![CDATA[shell> cd pgsphere-X.X.X]]>
+  </programlisting>
+
+  <para>
+   Compile the code. By default, &pgsphere; is compiled with the HEALPIX
+   support.
+  </para>
+  <programlisting>
 <![CDATA[shell> make]]>
-            </programlisting>
-            <para>
-                and to install <application>pgSphere</application> :
-            </para>
-            <programlisting>
+  </programlisting>
+  <para>or compile without HEALPIX support:</para>
+  <programlisting>
+<![CDATA[shell> make USE_HEALPIX=0]]>
+  </programlisting>
+
+  <para>
+   Run regression tests optionally. If the &pgsphere; was compiled without
+   HEALPIX support, USE_HEALPIX=0 should be specified in make command line.
+  </para>
+  <programlisting>
+<![CDATA[shell> make test]]>
+  </programlisting>
+
+  <para>
+   Install &pgsphere; files to the installation directories. The installation
+   directories are defined by &pg_config; utility. Superuser (root) access
+   rights may be required.
```

Comment:
I would recommend adding the setence "If &pgsphere; was compiled without HEALPix support, USE_HEALPIX=0 should be added after make in the following command."

Reply:
I thought that make install is enough and adding USE_HEALPIX=0 install is not necessary. There are some cases when developer compiles and installs the library using make install command only. In this case USE_HEALPIX should be added to disable the healpix support.

Reply:
I think that is incorrect and you must specify `USE_HEALPIX=0` when you `make install`, but, if you have tried it and it works, then I'll take your word for it. I'm pretty sure I tried it once and it caused problems, but I might be remembering `make test` or another target.

Reply:
@esabol You are right, USE_HEALPIX=0 should be added to make install. Otherwise, it will compile moc.c and process_moc.cpp. Anyway, I've modified the doc as you suggested. Thank you!